src/docfx/build/xref/XrefMap.cs

```diff
@@ -156,20 +153,49 @@ private static XrefSpec LoadSchemaDocument(List<Error> errors, JToken token, Doc
 
                 var (schemaErrors, content) = JsonUtility.ToObject(token, schema.Type, transform: TransformContent);
                 errors.AddRange(schemaErrors);
-                return xref;
+                return xrefs.Values;
 
-                object TransformContent(DataTypeAttribute attribute, object value)
+                object TransformContent(DataTypeAttribute attribute, object value, string jsonPath, string fieldName)
                 {
                     if (attribute is UidAttribute)
                     {
-                        if (!string.IsNullOrEmpty((string)value))
+                        var level = GetLevelFromPath(jsonPath);
+                        if (!xrefs.ContainsKey(level))
+                        {
+                            xrefs.Add(level, new XrefSpec { Uid = (string)value, Href = file.SitePath });
+                        }
+                        else
+                        {
+                            errors.Add(Errors.MultipleUid());
+                        }
+                    }
+
+                    if (attribute is XrefPropertyAttribute)
+                    {
+                        if (!xrefs.TryGetValue(GetLevelFromPath(jsonPath), out var xref))
+                        {
+                            errors.Add(Errors.UidMissing());
```

Comment:
>Errors.UidMissing() [](start = 39, length = 19)

Also feels this is a schema error. In document, missing uid is acceptable if users don't need xref reported.

Reply:
Same reason as above